tests/HashMapProperties.hs

```diff
@@ -49,6 +49,14 @@ pFoldable :: [(Int, Int)] -> Bool
 pFoldable = (L.sort . Foldable.foldr (:) []) `eq`
             (L.sort . Foldable.foldr (:) [])
 
+pHashable :: [(Key, Int)] -> [Int] -> Int -> Property
+pHashable xs is salt =
+    x == y ==> hashWithSalt salt x === hashWithSalt salt y
```

Comment:
Correct.

QuickCheck doesn't gave up for me, yet I could change this to
```hs
pHashable zs is salt =
    x == y ==> hashWithSalt salt x === hashWithSalt salt y
  where
    xs = Map.fromList . Map.toList $ zs -- Drop duplicate keys
    ys = shuffle is xs
```

Then `x == y ==>` won't be strictly required, but i'd keep as it communicates the intention.